Microsoft.Toolkit.Uwp.Services/Services/OneDrive/OneDriveStorageFile.cs

```diff
@@ -85,6 +88,31 @@ public OneDriveStorageFile(IOneDriveClient oneDriveProvider, IItemRequestBuilder
         }
 
         /// <summary>
+        /// Creates a background download for the current file
+        /// </summary>
+        /// <param name="destinationFile">A <see cref="StorageFile"/> to which content will be downloaded</param>
+        /// <param name="completionGroup">The <see cref="BackgroundTransferCompletionGroup"/> to which should <see cref="BackgroundDownloader"/> reffer to</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request</param>
+        /// <returns>The created <see cref="DownloadOperation"/></returns>
+        public async Task<DownloadOperation> CreateBackgroundDownloadAsync(StorageFile destinationFile, BackgroundTransferCompletionGroup completionGroup = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (destinationFile == null)
+            {
+                throw new ArgumentNullException(nameof(destinationFile));
+            }
+
+            var requestMessage = Provider.Drive.Items[OneDriveItem.Id].Content.Request().GetHttpRequestMessage();
+            await Provider.AuthenticationProvider.AuthenticateRequestAsync(requestMessage).AsAsyncAction().AsTask(cancellationToken);
+            var downloader = completionGroup == null ? new BackgroundDownloader() : new BackgroundDownloader(completionGroup);
+            foreach (var item in requestMessage.Headers)
+            {
+                downloader.SetRequestHeader(item.Key, item.Value.First());
+            }
+
+            return await Task.Run(() => downloader.CreateDownload(requestMessage.RequestUri, destinationFile), cancellationToken);
```

Comment:
No its okay. I don't think Task.Run will help there. We can look at it if there is a perf issue.

ConfigureAwait might help here
var requestMessage = Provider.Drive.Items[oneDriveId].Content.Request().GetHttpRequestMessage(); 

await Provider.AuthenticationProvider.AuthenticateRequestAsync(requestMessage).AsAsyncAction().AsTask(cancellationToken);